src/storage/mongodb/BillingStorage.ts

```diff
@@ -234,4 +237,61 @@ export default class BillingStorage {
     // Debug
     await Logging.traceEnd(tenantID, MODULE_NAME, 'deleteInvoice', uniqueTimerID, { id });
   }
+
+  // ------------------------------------------------------------------------------------------
+  // BILLING SETTINGS
+  // ------------------------------------------------------------------------------------------
+  public static async getBillingSettings(tenantID: string): Promise<BillingSettings[]> {
+    const settings = await SettingStorage.getSettings(tenantID, { identifier: TenantComponents.BILLING }, Constants.DB_PARAMS_SINGLE_RECORD);
```

Comment:
Add the debug methods in all Storage public methods to trace the DB calls like in other storages.

Reply:
Yes indeed if the method will delegate to SettingStorage, no need to put debug here.